Cellular/emodels/iv_fi_curve/analysis_info.json

```
@@ -20,6 +20,7 @@
         "NEURON>=8.0.2,<9.0.0",
         "entitysdk>=0.5.0",
         "obi-auth>=0.3.0",
-        "ipywidgets>=8.1.0"
+        "ipywidgets>=8.1.0",
+        "obi_notebook"
```

maybe [obi-notebook](https://pypi.org/project/obi-notebook/)
```suggestion
        "obi-notebook>=0.1.dev10"
```